docs/reference/eql/detect-threats-with-eql.asciidoc

```diff
@@ -0,0 +1,424 @@
+successfully.
++
+[source,console]
+----
+GET /_cat/indices/my-index-000001?v&h=health,status,index,docs.count
+----
+// TEST[setup:atomic_red_regsvr32]
++
+The API returns the following response. The `docs.count` value should show `150`
+documents were indexed.
++
+[source,txt]
+----
+health status index            docs.count
+yellow open   my-index-000001         150
+----
+// TESTRESPONSE[non_json]
+
+[discrete]
+[[eql-ex-get-a-count-of-regsvr32-events]]
+=== Get a count of `regsvr32.exe` events
+
+Since you're looking for `regsvr32` misuse, start by getting a count of events
+using `regsvr32.exe`.
+
+The following <<eql-search-api,EQL search API>> request uses an EQL query to
+retrieve a count of events with a `process.name` of `regsvr32.exe`. The query
+starts with the <<eql-syntax-match-any-event-category,`any where` keywords>>,
+meaning the query can match events of any <<eql-required-fields,event
+category>>.
+
+[source,console]
+----
+GET /my-index-000001/_eql/search?filter_path=-hits.events  <1>
+{
+  "query": """
+    any where process.name == "regsvr32.exe"        <2>
+  """,
+  "size": 150                                       <3>
+}
+----
+// TEST[setup:atomic_red_regsvr32]
+
+<1> Uses the `?filter_path=-hits.events` query parameter to exclude the
+`hits.events` property from the response. The `hits.events` property contains
+the document source for any matching events. This request is intended to
+retrieve a count of events only.
+<2> Uses an EQL query to match events with a `process.name` of `regsvr32.exe`.
+<3> Returns up to 150 events or sequences matching the EQL query.
+
+The request returns the following response, indicating that 143 events match the
+query.
+
+[source,console-result]
+----
+{
+  "is_partial": false,
+  "is_running": false,
+  "took": 60,
+  "timed_out": false,
+  "hits": {
+    "total": {
+      "value": 143,
+      "relation": "eq"
+    }
+  }
+}
+----
+// TESTRESPONSE[s/"took": 60/"took": $body.took/]
+
+[discrete]
+[[eql-ex-check-for-command-line-artifacts]]
+=== Check for command line artifacts
+
+Your previous query matched 143 events. Many of these events are likely to be
+false positives.
+
+To narrow down the results, check if the command line was used during any of
+these events. Append `and process.command_line > 0` to the previous EQL query.
+This limits matching results to events that contain a `process.command_line`
+value.
+
+You'll also need to remove the `filter_path=-hits.events` query parameter. This
+lets you retrieve the document source for any matching events.
+
+[source,console]
+----
+GET /my-index-000001/_eql/search
+{
+  "query": """
+    any where process.name == "regsvr32.exe" and process.command_line > 0
+  """,
+  "size": 150
+}
+----
+// TEST[setup:atomic_red_regsvr32]
+
+The query matches one event. Based on the event's `process.command_line` value,
+`regsvr32.exe` was used to register a script as `scrobj.dll`. This fits known
+regsvr32 misuse behavior.
```

Comment:
This specific variant is called "Squibblydoo." You don't have to include that name, but another pointer back to https://attack.mitre.org/techniques/T1218/010/ could be good here, if you want to "read on for more information"  about how this works.